### ISSUE TITLE
Beta feedback

### ISSUE THREAD

**Issue by jjpeet** [home-assistant/home-assistant-iOS#154]
Hi, @robbiet480 thanks for your hard work on this one. I'd lend a hand if I knew how to code! So here's my first impressions of the beta. I'm not sure if this is the right place to report it all, but here goes;

I just downloaded the beta which replaced the latest released version on my iPhone X and I can report the following;
I'm using HA 0.78.3
I've done a reset (by clicking the reset button in the app) and disabled notifications and location access. I did this because the settings brought forward from the old app didn't seem to work (x marks against notifications & location)

After resetting, killing the app and starting again I tried the following;
1. Put in my URL
2. Turned on Legacy Auth and entered the password.
3. Clicked save
The outcome was;
1. I get an x against iOS Component Loaded
2. I then tried enabling location tracking - nothing happened.
3. I then tried enabling notifications - nothing happened.
4. When I clicked done, it says "unable to connect to Home Assistant" RETRY.
The event log is clear.

Now if I kill the app and re-open;
1. I get a tick against iOS Component Loaded
2. Still no prompts after clicking enable location tracking and notifications
3. Clicking done retains the same "Unable to connect to Home Assistant" message
4. Event log clear

Next attempt: Turned off legacy auth and turned on basic auth (and entered u/p)
1. Clicked save and it visited the HASS web page and prompted for username and password (the screen you get on first time use of Hass on a browser).
2. I entered my u/p and it returned to the settings screen.
3. Tried again to enable location and notifications - nothing.
Event log says: 
1. Request(SSID: Unavailable myurl/api/ios/identify
2. Request (SSID: Unavailable myurl/api/states
3. Request (SSID: Unavailable myurl/manifest.json
4. Request (SSID: Unavailable myurl/api/config

So I turned on Use legacy auth (as well as basic auth), and no extra errors appearing.
Still can't enable location or notifications
Clicking done gets me to the "Unable to connect to Home Assistant" error message again.

So that's my first pass using the beta - hope this info helps.
In short, I'd say;
1. there could be a little more guidance in the settings interface as to what I should be turning on and off (in relation to auth options)
2. Looks like a snafu in there somewhere in relation to getting the Apple box to appear to enable notifications and location
3. Maybe a way to clear the browser cache to allow me to re-enter my for HASS password? I suspect that the reset function isn't clearing the cache.
4. Maybe a function to email the logs?

Thx
JP

**Comment by robbiet480** [comment]
2 things:

1. Please replace all the `.` you have with numbers so it's properly Markdown formatted and so much easier to read 😄
2. Are you on the very latest beta? It's 1.2.0 (35) off the top of my head (the 35 bit is what really matters). Shortly after announcing open beta signups I pushed another release live so it's possible you missed the update.

**Comment by jjpeet** [comment]
OK, replaced the dots :)

Did a reboot of my phone and now the Hass interface loads...

Then I saw your reply and then did the update to 35. I can report the following;
1. The Use legacy auth switch stays off (even though I turned it on and hit save)
2. Still no location or notification box coming up, so I manually turned them on in iOS settings, and I not get three ticks...

Hope that helps,
JP

**Comment by jjpeet** [comment]
In addition, none of my notifications are coming through on the new app.

It looks like it's the same device ID in known-devices.yaml, and the same device tracker entity is being updated with location and batter data etc (device_tracker.jamess_iphone)

I do remember some potential changes I needed to make to my notification settings? Maybe that's it...

I'm getting the following in my Home-assistant.log;
`The target (['4b90cb08-1be1-31ce-bc44-22c0e6d0bf72']) does not exist in .ios.conf.`

JP

**Comment by robbiet480** [comment]
@jjpeet it's been a few days and few more versions, how are things?

**Comment by jjpeet** [comment]
Hi @robbiet480 it's certainly getting pretty solid now. I just downloaded the latest, and the only issue I'm seeing now is that in setting, when I see crosses (often due to the fact that my work wifi doesn't allow port 8123, I will turn off wifi, and click Validate and save connection settings, but don't get the ticks unless I kill the app  and restart it...

Notifications are now working fine (inc images),

Top job.....

JP

**Comment by jjpeet** [comment]
Hi @roflmao I’m not too sure to be honest.
My son just setup a new phone and required the legacy auth to be on. I’m just looking at my settings and legacy auth is turned off - both my and my son’s app is working...

What are you finding?
JP

**Comment by robbiet480** [comment]
@jjpeet How's it looking?

**Comment by jjpeet** [comment]
Hi @robbiet480  All working great except for 
https://github.com/home-assistant/home-assistant-iOS/issues/194

And I’ve now settled on NOT turning on legacy auth and everything is fine..

Thx
JP

**Comment by robbiet480** [comment]
Great! Gonna close this and leave #194 open
